wkcuber/compress.py

```diff
@@ -46,6 +47,8 @@ def create_parser():
         "--mag", "-m", nargs="*", help="Magnification level", default=None
     )
 
+    parser.add_argument("--verify", default=False, action="store_true")
```

Comment:
What happens with this flag? Can't find any use for it?

Also, please add a description.

Reply:
Sorry that is an obsolete artifact. I removed it again.